CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Grafana Mimir
 
 * [CHANGE] Compactor: delete source and output blocks from local disk on compaction failed, to reduce likelihood that subsequent compactions fail because of no space left on disk. #2261
+* [CHANGE] Ruler: Remove unused fields `-ruler.search-pending-for` and `-ruler.flush-period`. #2288
```

Comment:
[nit] Per docs squad, we don't call them "fields" anymore but we're specific whether they're "CLI flags" or "YAML config options".

```suggestion
* [CHANGE] Ruler: Remove unused CLI flags `-ruler.search-pending-for` and `-ruler.flush-period` (and their respective YAML config options). #2288
```

_Going to auto-apply this edit._